resources/scripts/components/server/startup/StartupContainer.tsx

```diff
@@ -124,7 +124,7 @@ const StartupContainer = () => {
                     )}
                 </TitledGreyBox>
             </div>
-            <h3 css={tw`mt-8 mb-2 text-2xl`}>Variables</h3>
+            {data.variables.length != 0 && <h3 css={tw`mt-8 mb-2 text-2xl`}>Variables</h3>}
```

Comment:
Please use `> 0`.  Also, if you are ever writing comparisons in the future, please use type-strict comparisons (`!==`, `===`, etc)